resources/build/bootstrap-ubuntu-20.04.sh

```diff
@@ -5,7 +5,7 @@
 set -xeo pipefail
 sudo apt-get update
 # Install gcc, clang-tidy and Python 3.
-sudo apt-get install -y build-essential clang-tidy-12 python3-pip
+sudo apt-get install -y build-essential pkg-config clang-format-12 clang-tidy-12 python3-pip
```

Comment:
[consider][minor] Install `pkgconf` instead, since its apparently better and actively maintained, e.g. from [Fedora changelog](https://fedoraproject.org/wiki/Changes/pkgconf_as_system_pkg-config_implementation#Detailed_Description)

> pkgconf is a newer, actively maintained implementation of pkg-config that supports more aspects of the pkg-config file specification

I tested it and installing `pkgconf` does work in the Vagrant box.

Reply:
Good call, thanks. Updated [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/429/commits/5be2d2cce999fa17d43c17eadb565120811a18db).